tests/scenes/biosphere/test_leaf_cloud.py

```diff
@@ -42,7 +42,7 @@ def tempfile_leaves():
 
 def test_inversebeta(rng):
     """Unit tests for :func:`_inversebeta`."""
-    assert np.allclose(_inversebeta(1, 1, rng), 0.4178246)
+    assert np.allclose(_inversebeta(1, 1, rng), 0.914478941)
```

Comment:
Basing this test on a RNG seems fragile to me (in the unlikely event of a RNG update, it will break while it should not). I'd rather have something more robust, properly checking that the `_inversebeta()` function really does what it is supposed to (*i.e.* approximate a more complicated thing).